src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -504,6 +505,49 @@ public bool IsIronPythonDialogDisabled
             }
         }
 
+
+        private DynamoPythonScriptEditorTextOptions editTextOptions = new DynamoPythonScriptEditorTextOptions();
+        /// <summary>
+        /// Gets/Sets the text editor options for python script editor.
+        /// </summary>
+        public DynamoPythonScriptEditorTextOptions PythonScriptEditorTextOptions
```

Comment:
hmm!
so - if the binding is no longer needed...
1. why is this public?
2. why are we raising a property changed event?

Reply:
I will remove the `set` block altogether as it is never being called.